README.md

```diff
@@ -3,31 +3,52 @@
 [![Puppet Forge](https://img.shields.io/puppetforge/v/simp/simp_core.svg)](https://forge.puppetlabs.com/simp/simp_core)
 [![Puppet Forge Downloads](https://img.shields.io/puppetforge/dt/simp/simp_core.svg)](https://forge.puppetlabs.com/simp/simp_core)
 [![Puppet Forge Modules](https://img.shields.io/puppetforge/mc/simp.svg)](https://forge.puppetlabs.com/simp)
-[![Build Status](https://travis-ci.org/simp/simp-core.svg)](https://travis-ci.org/simp/simp-core)
 
 This is the [SIMP](https://simp-project.com) supermodule
 
 The latest release is [![GitHub release](https://img.shields.io/github/release/simp/simp-core.svg)](https://github.com/simp/simp-core/releases)
 
 You can find all full SIMP changelogs on [Read The Docs](https://simp.readthedocs.io/en/master/changelogs)
 
-1. [Supported releases](#supported-releases)
-2. [Getting Started](#getting-started)
-3. [Relevant Links](#relevant-links)
+<!-- vim-markdown-toc GFM -->
 
+* [Supported Releases](#supported-releases)
+* [Should Work but Less Tested](#should-work-but-less-tested)
+* [In Progress](#in-progress)
+* [Getting Started](#getting-started)
+* [Commercial Support](#commercial-support)
+* [Relevant Links](#relevant-links)
+
+<!-- vim-markdown-toc -->
 
 ## Supported Releases
 
-  * [Red Hat Enterprise Linux 7.5](https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/7.5_Release_Notes/index.html)
-  * [CentOS 7 1804](https://wiki.centos.org/Manuals/ReleaseNotes/CentOS7.1804)
-  * [Red Hat Enterprise Linux 6.10](https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/6/html/6.10_Release_Notes/index.html)
-  * [CentOS 6.10](https://wiki.centos.org/Manuals/ReleaseNotes/CentOS6.10)
+* [Red Hat Enterprise Linux 8](https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/8)
+* [Red Hat Enterprise Linux 7](https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7)
+* [CentOS Stream 8](https://www.centos.org/centos-stream/)
+* [CentOS 7](https://wiki.centos.org/Manuals/ReleaseNotes/CentOS7)
+* [Oracle Linux 7](https://docs.oracle.com/en/operating-systems/oracle-linux/7/)
+* [Oracle Linux 8](https://docs.oracle.com/en/operating-systems/oracle-linux/8/)
+
+## Should Work but Less Tested
+
+* [Rocky Linux 8](https://rockylinux.org/)
+* [Alma Linux 9](https://almalinux.org/)
+
```

Comment:
Unless users do something to prevent `assert_metadata` from failing, this won't work

Reply:
Booo...I thought they had gotten into enough places